src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/ObjectConverter.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.ons.ctp.response.collection.exercise.endpoint;
```

Comment:
Having this be in the endpoint folder feels wrong as this doesn't deal with http requests.  Maybe in domain or representation?

Reply:
You are right. Thinking about it now it shouldn't be here. I've moved it to domain as suggested. It's more relevant.